### PR TITLE
1066 add project model and sync to django autotask

### PR DESCRIPTION
**Pull request by kti-cameron**

### CODE REVIEW

File: djautotask/models.py [hunk 1]
```diff
@@ -52,13 +71,103 @@ class Picklist(TimeStampedModel):
...
+class Project(TimeStampedModel):
+    name = models.CharField(max_length=100)
+    number = models.CharField(null=True, max_length=50)
+    description = models.CharField(max_length=2000)
+    actual_hours = models.DecimalField(
+        null=True, decimal_places=2, max_digits=9)
+    completed_date_time = models.DateTimeField(null=True)
+    completed_percentage = models.PositiveSmallIntegerField(default=0)
+    duration = models.PositiveSmallIntegerField(default=0)
+    start_date_time = models.DateTimeField(null=True)
```
Comment (kti-sam) [inline djautotask/models.py]: Huh. They really want to track their projects closely I guess. "THE PROJECT WILL START FRIDAY AT EXACTLY 2:54 PM! AND NOT A MINUTE SOONER!"

Comment (kti-sam) [reply djautotask/models.py]: Actually no, its not. I checked autotask and it looks like they display it as a date only and ignore the time even though it comes in as a date time. We can probably store it as a date field too then just like we do in djconnectwise. Unless there is a good reason not to. I feel like ambiguous project start/end dates based on time zones is not good though.

File: djautotask/models.py [hunk 2]
```diff
@@ -52,13 +71,103 @@ class Picklist(TimeStampedModel):
...
+
+
+class ProjectStatus(Picklist):
+    pass
+
+    class Meta:
+        verbose_name_plural = 'Project statuses'
+
+
+class ProjectType(Picklist):
```
Comment (kti-sam) [inline djautotask/models.py]: Now this and status were not technically part of this issue, but I can't think of a good enough reason to say take it out at this point... so :woman_shrugging:

Comment (kti-cameron) [reply djautotask/models.py]: Yeah I know. I'm guilty. I figured they'll be useful later on and they were super easy to add while I was doing everything else.

File: djautotask/models.py (hunk 1 above)
Comment (kti-cameron) [reply djautotask/models.py]: Good point. Might as well take the same approach as django-connectwise. :+1:

**Comment by kti-cameron** [comment]
Alright, resolved the conflicts. Ready for another look

**Comment by kti-cameron** [comment]
Actually not ready yet. Still need to fix some sync date parse issues.

**Pull request merged**